### PR TITLE
Create update page update/styling

### PR DESCRIPTION
**Pull request by Skarrg**

### CODE REVIEW

**Review by BowArrow — APPROVED** [review]
Looks good, just some deletes and replacements

File: src/Components/CreateUpdateBounty/style.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+.bountytext {
+    width: 80vh;
+    height:20vh;
+}
+.rewardtext {
+    width: 80vh;
+    height:10vh;
+}
+.bountylocation {
+float: left
```
Comment (DChan0319) [inline src/Components/CreateUpdateBounty/style.css:14]: can you indent this right here

File: src/Components/CreateUpdateBounty/index.js [hunk 2]
```diff
@@ -30,49 +31,76 @@ class CreateUpdateBounty extends Component {
     render() {
```
Comment (DChan0319) [inline src/Components/CreateUpdateBounty/index.js:31]: This component's render function has a lot of small little components that could be extracted out to their own smaller components. For example, your div component for bounty title, bounty info, and bounty reward are all more or less the same. The only difference between all of them is just the text and labeling. Is there a way to make all these 1 component and then dynamically create this component with the text that you need and want?

**Comment by DChan0319** [comment]
Overall i don't see any major issues. Please read the comments i wrote. You don't necessarily have to make any big changes, but just to get you thinking about breaking down the component even more.

**Pull request closed without merging**